### DISCUSSION TITLE
Introducing a new, heavy dependency

### DISCUSSION THREAD

**Discussion by wrobelda** [Homebrew/discussions#848] [Tap maintenance and Homebrew development]
I am a maintainer of KMyMoney, of which brew formulae are not in Homebrew repo yet. We depend on qt5 GUI backend in aqbanking/gwenhywfar, both of which _are_ already in Homebrew.

At some point in time all the additional GUIs were removed from the formula ([1], [2]). There's no information why this happened – I assume it's because both gtk2 and qt4 were outdated, although they were also marked as ":optional". My understanding is that `:optional` (and `:recommended`) are currently not allowed in Homebrew/homebrew-core.

I want to add qt5 support, however, so that KMyMoney works as expected. Is it OK or is it this too big of a (non-optional) dependency? How to proceed here?

[1] https://github.com/Homebrew/homebrew-core/commit/325bdeb4d41c6d7cbf17ab6433cb9588ca277186#diff-cb11083137f0e460288c66a5101ba82af15f78cee1fe430921379f2154fd5f5e
[2] https://github.com/Homebrew/homebrew-core/commit/36e7c3aaea09a5f8d70fa7f1f49da5da1b08c192#diff-cb11083137f0e460288c66a5101ba82af15f78cee1fe430921379f2154fd5f5e

**Comment by SMillerDev** [comment]
There is one way to find out. Make a pull request and see what the maintainers say.
